services/debugger/destination/cache.go

```diff
@@ -0,0 +1,108 @@
+package destinationdebugger
+
+import (
+	"fmt"
+	"github.com/rudderlabs/rudder-server/config"
+	"sync"
+	"time"
+)
+
+var (
+	CacheTTL     time.Duration
+	CacheMaxSize int
+)
+
+func loadCacheConfig() {
+	config.RegisterIntConfigVariable(2, &CacheMaxSize, true, 1, "DestinationDebugger.maxEventsCacheSize")
+	config.RegisterDurationConfigVariable(720 * time.Hour, &CacheTTL, true, 1, "DestinationDebugger.maxEventsCacheTTL")
+}
+
+type cacheItem struct {
+	objs       []*DeliveryStatusT
```

Comment:
can we rename objs to deliveryEvents?

Reply:
changed to data, keeping it more generic